src/java_bytecode/generic_arguments_name_buildert.cpp

```diff
@@ -0,0 +1,37 @@
+/*******************************************************************\
+
+ Module: Generic Arguments Name Builder - A class to aid in building the name
```

Comment:
Normally this is a short name (probably Generic Arguments Name Builder) then there is a `/// file` comment below giving the more verbose description